### ISSUE TITLE
Тестирование поля глобального поиска

### ISSUE THREAD

**Issue by SeiOkami** [Pr-Mex/vanessa-automation#2106]
1. Версия Vanessa Automation: 1.2.041.1
   2. Версия платформы 8.3.24.1342
   3. Режим совместимости базы, где запускается Vanessa Automation: Версия 8.3.21
   4. Операционная система: Windows

Работа с полем глобального поиска не записывается. Я предполагаю, что платформа в принципе не имеет такой возможности, но на всякий случай регистрирую предложение о доработке Vanessa

Какая цель: 
У меня есть фреймворк, весь интерфейс которого находится в поле глобального поиска. Выходит, что инструмент не поддаётся тестированию =(
![image](https://github.com/Pr-Mex/vanessa-automation/assets/42138875/bfd9458c-38f6-4b76-859c-7c2f2ad56c0b)

**Comment by Pr-Mex** [comment]
@SeiOkami 
Да, сейчас штатно через АПИ автотестов с ним работать нельзя. Но можно через UI Automation.
